<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateExportFile</class>
 <widget class="QWidget" name="CreateExportFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建文件</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文件名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目录：</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnCancle">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>188</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreateFile">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>创建</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnChooseDir">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/Resources/folder.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fileDir">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>301</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>正在创建一个新的Excel文件...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BidsAssistant.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CreateExportFile</receiver>
   <slot>fileNameInputed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnChooseDir</sender>
   <signal>clicked()</signal>
   <receiver>CreateExportFile</receiver>
   <slot>btnChooseDirClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancle</sender>
   <signal>clicked()</signal>
   <receiver>CreateExportFile</receiver>
   <slot>btnCancleClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCreateFile</sender>
   <signal>clicked()</signal>
   <receiver>CreateExportFile</receiver>
   <slot>btnCreateFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileNameInputed()</slot>
  <slot>btnChooseDirClicked()</slot>
  <slot>btnCancleClicked()</slot>
  <slot>btnCreateFileClicked()</slot>
 </slots>
</ui>
